docs/concepts/pools/deploy-a-pool.md

```diff
@@ -0,0 +1,39 @@
+# Pool Creation
+
+## Overview
+
+**Note: Before creating a new pool, make sure there isn't already a similar pool. There is no advantage to fragmenting liquidity.**
+
+Anyone can create a pool on Balancer. WeightedPools can be deployed using the Pool creator Interface that can be accessed on [Mainnet](https://app.balancer.fi/#/ethereum), [Polygon](https://app.balancer.fi/#/polygon), [Arbitrum](https://app.balancer.fi/#/arbitrum), and [Gnosis Chain](https://app.balancer.fi/#/gnosis-chain). It's highly recommended that you deploy a pool on a testnet before deploying on a production network.
+
+## What Type of Pool Should I Deploy?
+
+The type of pool you deploy depends on your use case. After reading througt the descriptions of different Balancer pool types, decide which type fits best your use case.
```

Comment:
Perhaps provide a link to a portion of the docs where users can learn about different pool types.